packages/core/src/Stores/portfolio-store.js

```diff
@@ -408,6 +411,10 @@ export default class PortfolioStore extends BaseStore {
         if (isUserSold(contract_response)) this.positions[i].exit_spot = '-';
 
         this.positions[i].is_loading = false;
+
+        if (this.root_store.ui.is_mobile && getEndTime(contract_response)) {
+            this.root_store.notifications.addTradeNotification(this.positions[i].contract_info);
```

Comment:
Is there any chance this could be called in one of the reports pages?

Reply:
@balakrishna-deriv yes, and it could. I intended this in case a user sells a contract from Reports while he/she is watching the main DTrader page open in another tab. But I just realized that notification will appear anyway while the main DTrader page is open, so I'm gonna limit this to the main DTrader page only. Thank you :)